docs/guides/integration-mobile-client-server.md

```diff
@@ -108,15 +108,15 @@ At this point, you are ready to use the EUID Identity generated server-side in t
 
 ### Using the EUID Integration Environment
 
-By default, the SDK is configured to work with the EUID production environment: `https://prod.uidapi.com`. If you want to use the EUID integration environment instead, provide the following URL in your call to EUIDManager initialization:
+By default, the SDK is configured to work with the EUID production environment: `https://prod.euid.eu/v2`. If you want to use the EUID integration environment instead, provide the following URL in your call to EUIDManager initialization:
 
 <Tabs groupId="language-selection">
 <TabItem value='android' label='Android'>
 
 ```js
 EUIDManager.init(
   context = this,
-  serverUrl = "https://operator-integ.uidapi.com"
+  serverUrl = "https://integ.euid.eu/v2"
```

Comment:
@dcaunt Hi Dave, in the dev app, it has an [example](https://github.com/IABTechLab/uid2-android-sdk/blob/main/dev-app/src/main/java/com/uid2/dev/DevApplication.kt#L24-L27) that has been updated as below after merging the EUID support code.
```           
       EUIDManager.init(
                context = this,
                EUIDManager.Environment.Custom(EUID_INTEG_SERVER_URL),
                isLoggingEnabled = true,
        )
```
Do we need to update the code example in this doc as below?
```
       EUIDManager.init(
                context = this,
                EUIDManager.Environment.Custom("https://integ.euid.eu/v2")
        )
```